templates/mailhost/mailhost_ssl.erb

```diff
@@ -30,7 +30,7 @@ server {
   ssl_certificate_key  <%= @ssl_key %>;
   ssl_session_timeout  5m;
 
-  ssl_protocols TLSv1;
+  ssl_protocols <%= @ssl_protocols %>;
```

Comment:
The same change applies to the  puppet-nginx/templates/mailhost/mailhost.erb file. Because the SSL part of the two templates are equal I used to combine the two into one file that gets included. I'll fix the missing variable in my PR #769.